networkx/drawing/nx_pylab.py

```diff
@@ -715,11 +901,23 @@ def draw_networkx_edges(
     # FancyArrowPatch handles color=None different from LineCollection
     if edge_color is None:
         edge_color = "k"
-    edgelist_tuple = list(map(tuple, edgelist))
 
     # set edge positions
     edge_pos = np.asarray([(pos[e[0]], pos[e[1]]) for e in edgelist])
 
+    if len(edgelist[0]) == 3:
+        # MultiGraph input
+        edge_keys = [e[2] for e in edgelist]
+        max_multi_edges = max([e[2] for e in G.edges(keys=True)]) + 1
+        if len(connectionstyle) < max_multi_edges:
+            raise nx.NetworkXError(
+                f"{len(connectionstyle)} connectionstyle inputs provided"
+                "are not enough as maximum edges per node pair in MultiGraph"
+                "is {max_multi_edges}"
+            )
```

Comment:
I'm just going to try this shift where we let one connectionstyle through (all the errors in the documentation build give error in this case).
```suggestion
        if len(connectionstyle) > 1:
            max_multi_edges = max([e[2] for e in G.edges(keys=True)]) + 1
            if len(connectionstyle) < max_multi_edges:
                raise nx.NetworkXError(
                    f"{len(connectionstyle)} connectionstyle inputs provided "
                    "number of multiedges are not enough as maximum edges per node "
                    f"pair in MultiGraph is {max_multi_edges}"
            )
```